app/assets/stylesheets/style.scss

```diff
@@ -0,0 +1,135 @@
+/*font-family: 'Courgette', cursive;
+font-family: 'Amaranth', sans-serif;
+font-family: 'Nothing You Could Do', cursive;
+font-family: 'Berkshire Swash', cursive;*/
+
+// General Styling   //
+body {
+  
+}
+
+.background-image {
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: -1;
+
+  display: block;
+  background-image: image-url("HD-Wallpaper-Cat-With-Beer.jpeg");
+  background-color: #cccccc;
```

Comment:
Color literals like `#cccccc` should only be used in variable declarations; they should be referred to via variable everywhere else.